lib/tasks/cleanup.rake

```diff
@@ -14,24 +17,26 @@ namespace :abbyy do
 
       num_deleted += 1
       puts "Deleting ABBYY result file #{file}"
-      FileUtils.rm(file)
+      should_perform_deletions ? FileUtils.rm(file) : puts("(dry run) would have deleted #{file}")
     end
     puts "Deleted #{num_deleted} ABBYY result files older than 1 week."
     puts
 
     # Delete ABBYY input directories that are empty and ticket files older than 1 week
     input_entries = Dir.glob("#{Settings.sdr.abbyy.local_ticket_path}/*")
-    puts "Checking ABBYY input folders/files #{Settings.sdr.abbyy.local_ticket_path} for deletion: #{input_entries.size} folders found."
+    puts "Checking ABBYY input folders/files #{Settings.sdr.abbyy.local_ticket_path} for deletion: #{input_entries.size} folders/files found."
     num_deleted = 0
     input_entries.each do |entry|
       if File.directory?(entry) && Dir.empty?(entry)
         num_deleted += 1
         puts "Deleting empty ABBYY input directory #{entry}"
-        FileUtils.rm_rf(entry)
+        should_perform_deletions ? FileUtils.rm_rf(entry) : puts("(dry run) would have deleted #{entry}")
       elsif File.file?(entry) && File.mtime(entry) < Time.now - 1.week
         num_deleted += 1
         puts "Deleting ABBYY ticket file #{entry}"
-        FileUtils.rm_f(entry)
+        should_perform_deletions ? FileUtils.rm_f(entry) : puts("(dry run) would have deleted #{entry}")
+      else
+        puts "skipping #{entry} (if it's a directory, it wasn't empty; if it's a file, it was less than 1 week old)"
```

Comment:
happy to get rid of this `else` branch if it might be too noisy.  but thought it might be helpful for debugging, and figured the noise wasn't such an issue since the task is run manually for unusual situations, as opposed to e.g. regularly as a cron job.